RestartFailedTasks: correct the arg name (#36883)
demisto/content: Packs/IntegrationsAndIncidentsHealthCheck/ReleaseNotes/1_3_21.md

#### Scripts

##### RestartFailedTasks

> Fixed an issue where the **Restart Failed Tasks** script reopen also tasks that succeeded if they have the same task id as the failed task.

Reviewer: ```suggestion
Fixed an issue where the **Restart Failed Tasks** script incorrectly reopened successful tasks that share the same task ID with failed tasks.
```
